.github/workflows/tyk-headless.yaml

```diff
@@ -0,0 +1,52 @@
+name: Tyk Headless Push Helm
+on: push
+jobs:
+  push:
+    name: Tyk Headless Push CS
+    runs-on: ubuntu-latest
+    branches:
+      - main
```

Comment:
i think this should refer to master instead of main. Similar thing for rest of workflow files

Reply:
Once on master we'll be running it on every commit.